packages/icons/src/library/copy.js

```diff
@@ -5,7 +5,8 @@ import { SVG, Path } from '@wordpress/primitives';
 
 const copy = (
 	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-		<Path d="M20.2 8v11c0 .7-.6 1.2-1.2 1.2H6v1.5h13c1.5 0 2.7-1.2 2.7-2.8V8zM18 16.4V4.6c0-.9-.7-1.6-1.6-1.6H4.6C3.7 3 3 3.7 3 4.6v11.8c0 .9.7 1.6 1.6 1.6h11.8c.9 0 1.6-.7 1.6-1.6zm-13.5 0V4.6c0-.1.1-.1.1-.1h11.8c.1 0 .1.1.1.1v11.8c0 .1-.1.1-.1.1H4.6l-.1-.1z" />
+		<Path d="M20 8v11c0 .69-.31 1-.999 1H6v1.5h13.001c1.52 0 2.499-.982 2.499-2.5V8H20Z" />
```

Comment:
Multiple `path`'s in a single SVG are technically fine. But I still have on my wishlist, an proper build system for our icons, so we can feed a folder of minified SVGs into a grunt system and have these components built automatically rather than manually as they are now. [The build system for Gridicons](https://github.com/Automattic/gridicons/) requires there be only a single path in those minified SVGs. Since we don't yet have multi-colored or animated icons, I wonder if we should keep that tentative rule for now, and make icons here into single compound shapes?

Reply:
I use Illustrator's Expand capability:

![Screenshot 2023-11-28 at 09 12 25](https://github.com/WordPress/gutenberg/assets/1204802/b036a89a-222d-4c36-b26c-d3d22eb33646)

That gives me this:

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" style="enable-background:new 0 0 24 24" xml:space="preserve"><path d="M18 16V5c0-1.1-.9-2-2-2H5c-1.1 0-2 .9-2 2v11c0 1.1.9 2 2 2h11c1.1 0 2-.9 2-2zM4.5 16V5c0-.3.2-.5.5-.5h11c.3 0 .5.2.5.5v11c0 .3-.2.5-.5.5H5c-.3 0-.5-.2-.5-.5zM20 8v11c0 .7-.3 1-1 1H6v1.5h13c1.5 0 2.5-1 2.5-2.5V8H20z" /></svg>
```

It _should_ be possible in Figma as well, if you first make a union, and then "expand appearance" 🤔

Reply:
I don't have illustrator; but I've tried following that Figma flow and I still end up with multiple paths. Do one of you mind taking a look, or is that fine as a follow-up? 

I updated the relative icons in the library Figma.

Reply:
I redrew the icon and it seems to work now. I have no idea why :D 

```
<svg width="24" height="24" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M5 4.5h11a.5.5 0 0 1 .5.5v11a.5.5 0 0 1-.5.5H5a.5.5 0 0 1-.5-.5V5a.5.5 0 0 1 .5-.5ZM3 5a2 2 0 0 1 2-2h11a2 2 0 0 1 2 2v11a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5Zm17 3v10.75c0 .69-.56 1.25-1.25 1.25H6v1.5h12.75a2.75 2.75 0 0 0 2.75-2.75V8H20Z" /></svg>
```